app/Http/Controllers/ChopsController.php

```diff
@@ -138,30 +141,24 @@ public function show($id)
 	 */
 	public function edit($id)
 	{
-		//
-	}
+        $owner_id = Chop::find($id)->user_id;
+        $user = Auth::user();
```

Comment:
Assign `Chop::find($id)` to a variable and use that variable instead to avoid duplication.